style.css

```diff
@@ -456,21 +493,51 @@
   margin-bottom: 1rem;
 }
 
+.close {
+  position: absolute;
+  top: 42px;
+  left: 5%;
+  width: 20px;
+  height: 20px;
+  cursor: pointer;
+}
+
 @media (max-width: 767px) {
```

Comment:
- [ ] The pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design, kindly make sure to use these colors in your pages `#272A31`, `#EC5242`, `#D3D3D3`. I can see that you only used `#EC5242`